### PR TITLE
Dr patterns january

### PR DESCRIPTION
**Pull request by klaudiarybicka**
I implemented the changes Daniela asked for. The pages to consult are "Notices" and "Force of law", also to inform about the home page "Get help" section.

### CODE REVIEW

File: src/hmrc-content-guide/forms-detailed-guide/index.njk [hunk 1]
```diff
@@ -5,56 +5,99 @@ layout: twoColumnPage.njk
...
+
+<h3 class="govuk-heading-m">Open Document Formats</h3>
+
+<p class="govuk-body">
+If the form is in Open Document Format it will not require an accessibility statement.
+</p>
+
+<p class="govuk-body">
+You must [use the publication box !@n markdown in
+Whitehall](<a href="https://www.gov.uk/guidance/how-to-publish-on-gov-
```
Comment (Jon-Rowe-HMRC) [inline src/hmrc-content-guide/forms-detailed-guide/index.njk:42]: Suggest this is just...

You must use the [publication box](https://www.gov.uk/guidance/how-to-publish-on-gov-uk/markdown#publication-box) !@n markdown in Whitehall.

File: src/hmrc-content-guide/forms-detailed-guide/index.njk [hunk 2]
```diff
@@ -5,56 +5,99 @@ layout: twoColumnPage.njk
...
+</p>
+
+<h3 class="govuk-heading-m">Print and post forms</h3>
+
+<p class="govuk-body">
+These will be hosted on <a href="https://public-online.hmrc.gov.uk/lc/content/" class="govuk-link">https://public-online.hmrc.gov.uk/lc/content/</a>.
+</p>
+
+<p class="govuk-body">
+If the form is a print and form, include the following markdown:
```
Comment (Jon-Rowe-HMRC) [inline src/hmrc-content-guide/forms-detailed-guide/index.njk:54]: Think this is a mistake on the source doc, and should be...

'If the form is print and post, include the following markdown:'

File: src/hmrc-content-guide/forms-detailed-guide/index.njk [hunk 3]
```diff
@@ -5,56 +5,99 @@ layout: twoColumnPage.njk
...
     })
   }}
 
-<h3 class="govuk-heading-m">Forms that are attachments</h3>
+  <h3 class="govuk-heading-m">PDF forms that can be completed on-screen</h3>
 
-<p class="govuk-body">
-If you’re going to attach a file, you must use the publication box !@n markdown.
+  <p class="govuk-body">
+  You must [use the publication box !@n markdown in
```
Comment (Jon-Rowe-HMRC) [inline src/hmrc-content-guide/forms-detailed-guide/index.njk:85]: Think simplify to...

You must use the [publication box]([https://www.gov.uk/guidance/how-to-publish-on-gov- uk/markdown#publication-box) !@n markdown in Whitehall to attach the PDF form.

File: src/hmrc-content-guide/forms-publication/index.njk [hunk 4]
```diff
@@ -6,47 +6,50 @@ layout: twoColumnPage.njk
...
 
 <p class="govuk-body">
-If there’s a form number you should include it. For example:
+Before you use this pattern, you should <a href="https://design.tax.service.gov.uk/hmrc-content-guide/forms-check/" class="govuk-link">check which document type to use</a>, to decide
+if you should use a detailed guide instead.
 </p>
 
 <p class="govuk-body">
-'Register your business for VAT if you’re distance selling into Northern Ireland using form VAT1A'
+For an example see [Register for VAT if you&#39;re distance selling into Northern
```
Comment (Jon-Rowe-HMRC) [inline src/hmrc-content-guide/forms-publication/index.njk:23]: Think simplify to just...

For an example see [Register for VAT if you're distance selling into Northern Ireland](https://www.gov.uk/government/publications/vat-application-for-registration- distance-selling-vat1a).

File: src/hmrc-content-guide/forms-publication/index.njk [hunk 5]
```diff
@@ -85,6 +79,17 @@ You’ll need to fill in the form fully before you can print it. You cannot save
     })
   }}
 
+  <p class="govuk-body">
+  You should check with the DFOM (Digital Forms and Outbound Mail)
+team if you need to include the following:
+</p>
+
+<p class="govuk-body">
```
Comment (Jon-Rowe-HMRC) [inline src/hmrc-content-guide/forms-publication/index.njk:87]: Think this should be in a code snippet

File: src/hmrc-content-guide/forms-publication/index.njk [hunk 6]
```diff
@@ -102,32 +107,69 @@ This form is interactive (one that you complete on screen) and you must use [Ado
...
       canCopy: true
     })
   }}
 
-<h4 class="govuk-heading-s">Address</h4>
+<h4 class="govuk-heading-s">Welsh translation</h4>
 
 <p class="govuk-body">
-If the address to send the form is not included on the form itself, include:
+If the form is not translated into Welsh, include:
```
Comment (Jon-Rowe-HMRC) [inline src/hmrc-content-guide/forms-publication/index.njk:127]: 'If the form is not translated into Welsh and is applicable in Wales, include:'

File: src/hmrc-content-guide/notices/index.njk [hunk 7]
```diff
@@ -10,22 +10,20 @@ layout: twoColumnPage.njk
 
 <h2 class="govuk-heading-l">When to use</h2>
 
-<p class="govuk-body">Use this pattern to design an excise, Soft Drinks Industry Levy or VAT notice that includes force of law - for examples see:</p>
+<p class="govuk-body">
+Use this pattern to design an excise, Soft Drinks Industry Levy or VAT notice that includes <a href="https://design.tax.service.gov.uk/hmrc-content-guide/force-of-law" class="govuk-link">force
```
Comment (Jon-Rowe-HMRC) [inline src/hmrc-content-guide/notices/index.njk:14]: strip out 'https://design.tax.service.gov.uk' so the link works across our environments

Comment (klaudiarybicka) [reply src/hmrc-content-guide/notices/index.njk:14]: I am not sure what it's supposed to look like instead?

Comment (Jon-Rowe-HMRC) [reply src/hmrc-content-guide/notices/index.njk:14]: href="/hmrc-content-guide/force-of-law/"

**Review by Jon-Rowe-HMRC — APPROVED** [review]

**Pull request merged**